### ISSUE TITLE
Backend logger doesn't respect configured log level

### ISSUE THREAD

**Issue by kardeiz** [archivesspace/archivesspace#1258]
There is some part of the backend that is logging at the `INFO` level, despite the fact that I have set the log level to `error`:

    AppConfig[:backend_log_level] = "error"

My log file is filled with lines like:

```
May 31, 2018 9:25:33 AM org.apache.solr.core.SolrCore execute
INFO: [collection1] webapp= path=/select params={q=(id:("\/repositories\/2\/resources\/25\/tree\/node_from_root_3061"))&csv.escape=\&csv.header=true&start=0&csv.encapsulator="&fq=-exclude_by_default:true&fq=publish:true&rows=1&wt=json&facet=true} hits=1 status=0 QTime=0
```

When our site is crawled, the log file increases by dozens of GB a day.

All the other loggers are logging correctly.

Thanks!

**Comment by cfitz** [comment]
That's actually not the backend, that's Solr. 
Probably the easiest way to tweak the solr logs is to just use an[ external solr](https://archivesspace.github.io/archivesspace/user/running-archivesspace-with-external-solr/) and disabling the one that comes with Aspace.
This will probably help with the out of memory issue your running into on the PUI, since Solr will have it's own memory space.

**Comment by kardeiz** [comment]
Okay, thanks. I'd strongly prefer not to run my own Solr here if I don't have to. Will the Solr instance launched by ArchivesSpace respect the `SOLR_LOG_LEVEL` env var if I set/export that during ArchivesSpace initialization?

By my "out of memory issue," are you referring to #1257? If so, I don't think that is OOM error (though I could be wrong).

**Comment by cfitz** [comment]
Yeah,  I actually haven't tried adjusting the Solr levels, as I usually just go with an external solr ( if the records are of the "larger" size ).
I'd be curious to hear if that works...

But yeah, #1257  was what I was referring to. I have seen the that error message get raised before the actual out of memory error gets raised. You can adjust Jetty's buffer size:
https://github.com/archivesspace/archivesspace/blob/master/common/config/config-defaults.rb#L330-L331
You might bump those up and see if that helps...

**Comment by kardeiz** [comment]
Thanks for pointing out those config variables! I'll try raising those values

**Comment by kardeiz** [comment]
The "embedded" Solr provided by ArchivesSpace does not respect the `SOLR_LOG_LEVEL` variable (it doesn't resolve this logging issue).

Would it be possible to make sure `SOLR_LOG_LEVEL` is passed along to the AS Solr, or add a config variable to `config.rb` to adjust the Solr log level?

**Comment by HSteeleLTS** [comment]
I don't know if this is still an issue for people since new versions of ASPace require you to host your own instance of Solr anyway, but for those of us on older versions of ASpace I thought I'd share the solution I found.

I change the Solr logging level at the server level in Java’s configurations for Apache Solr.  A solution to this was discussed here:  https://solr-user.lucene.apache.narkive.com/JU6tFY0v/logging-in-solr-embedded  

What I did was location our JRE settings folder on the server with this command from the root “/”
•	sudo find . -name "jre" -type d
Then looking in this folder I found the Java logging properties file, and edited it
•	sudo nano -c /etc/java/java-1.8.0-openjdk/java-1.8.0-openjdk-1.8.0.322.b06-2.el8_5.x86_64/lib/logging.properties
Per the above page, I added this line to the logging configurations
•	org.apache.solr.level = WARNING

After restart, the logs were now longer recording regular "INFO" level solr events that had created so much noise, and the logs remained small
